tests/test_config.py

```diff
@@ -249,6 +250,40 @@ def test_log_config_file(mocked_logging_config_module):
     )
 
 
+@pytest.fixture(params=[0, 1])
+def web_concurrency(request):
+    yield request.param
+    if os.getenv("WEB_CONCURRENCY"):
+        del os.environ["WEB_CONCURRENCY"]
+
+
+@pytest.fixture(params=["127.0.0.1", "127.0.0.2"])
+def forwarded_allow_ips(request):
+    yield request.param
+    if os.getenv("FORWARDED_ALLOW_IPS"):
+        del os.environ["FORWARDED_ALLOW_IPS"]
+
+
+def test_env_file(web_concurrency: int, forwarded_allow_ips: str, caplog, tmpdir):
```

Comment:
I would use a `tmp_path` fixture instead, I'm not against the tmpdir usage though it's just that I feel it's cleaner to use Path-like objects.
I'll approve and let you choose !

Reply:
thanks